### PR TITLE
Monitor unknown attributes.

### PR DESCRIPTION
**Pull request by jimfb**
Warn unknown attributes.  Duplicates are handled by the deduping logic higher up in the same function.

### CODE REVIEW

**Comment by jsfb** [comment]
cc @sebmarkbage

File: src/addons/transitions/ReactCSSTransitionGroup.js [hunk 1]
```diff
@@ -57,9 +57,16 @@ var ReactCSSTransitionGroup = React.createClass({
   },
 
   render: function() {
+    var {
+      transitionName,
+      transitionAppear,
+      transitionEnter,
+      transitionLeave,
+      ...transferredProps
+    } = this.props;
```
Comment (sebmarkbage) [inline src/addons/transitions/ReactCSSTransitionGroup.js]: This will be slow since our transpiler isn't quite optimal here. :/

**Comment by sebmarkbage** [comment]
@zpao Will this lead to unused variables warnings?

File: src/addons/transitions/ReactCSSTransitionGroup.js (hunk 1 above)
Comment (jsfb) [inline src/addons/transitions/ReactCSSTransitionGroup.js]: Do we care?  Is someone fixing the transpiler soon, or do you want me to use Object.assign and manually remove the object properties?

Comment (sebmarkbage) [inline src/addons/transitions/ReactCSSTransitionGroup.js]: I mostly just care about the linting.

**Comment by sebmarkbage** [comment]
We need to figure out lint here first.

**Comment by jsfb** [comment]
Ok, @zpao says to "put it up, wait for him to get back from his meeting, no good answer at the moment."

**Comment by gaearon** [comment]
I’m closing because it’s been stale for a while, and there doesn’t appear to be consensus on how/whether to move forward with this. We can create a separate RFC issue if this is a part of a bigger plan but it’s hard to tell from a glance.

**Pull request closed without merging**